src/components/SideNav/SideNav.js

```diff
@@ -29,6 +29,10 @@ const ListItemEl = styled(ListItem)`
   margin: 0;
 `;
 
+/* Check to see if the parent directory has index.mdx */
+const checkIfOverview = (relativePath) =>
+  relativePath.split("/")[relativePath.split("/").length - 1] === "index.mdx";
```

Comment:
If this is getting broken out anyway, might be better to put `relativePath.split('/')` into a variable